### PR TITLE
Remove styles for frontend components

### PR DESCRIPTION
**Pull request by rvsia**
cc @Hyperkid123 @karelhala

### CODE REVIEW

File: src/sass/chrome.scss [hunk 1]
```diff
@@ -25,7 +25,7 @@ $pf-global--image-path: './assets/images/';
 
 @import '~@redhat-cloud-services/frontend-components/Spinner/spinner.css';
 @import '~@redhat-cloud-services/frontend-components/Skeleton/skeleton.css';
-@import '~@redhat-cloud-services/frontend-components/ConditionalFilter/onditionalFilter.css';
+@import '~@redhat-cloud-services/frontend-components/ConditionalFilter/conditional-filter.css';
```
Comment (Hyperkid123) [inline src/sass/chrome.scss:28]: This css should not be required anymore. Was the style broken without this?

Comment (rvsia) [reply src/sass/chrome.scss:28]: Nope, the link just throws 404.. should I remove it?

Comment (Hyperkid123) [reply src/sass/chrome.scss:28]: Yeah, try to remove it. The v3 components inject styling with components. This is theoretically rendundant. So is the stuff above.

Comment (rvsia) [reply src/sass/chrome.scss:28]: Should I remove styles for all frontend components?

Comment (rvsia) [reply src/sass/chrome.scss:28]: Yeah, I should. Let me try it.

Comment (rvsia) [reply src/sass/chrome.scss]: Looks OK locally.

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**